### PR TITLE
Attempting to allow the scripts to run with contexts

### PR DESCRIPTION
**Pull request by bbux-atg**
Acting as a proxy for @matthpeterson  to port changes from custom fork in order to sync up changes.

### CODE REVIEW

File: shell/src/main/java/org/apache/accumulo/shell/commands/ScriptCommand.java [hunk 1]
```diff
@@ -50,128 +55,172 @@
...
+    if (contextName != null && !contextName.equals("")) {
+      try {
+        AccumuloVFSClassLoader.getContextManager()
+            .setContextConfig(new ContextManager.DefaultContextsConfig() {
+              @Override
+              public Map<String,String> getVfsContextClasspathProperties() {
+                try {
+                  return shellState.getConnector().instanceOperations().getSystemConfiguration();
+                } catch (AccumuloException e) {
+                  throw new RuntimeException(e);
```
Comment (ctubbsii) [inline shell/src/main/java/org/apache/accumulo/shell/commands/ScriptCommand.java]: Should throw a more specific RTE, rather than just `RuntimeException`

**Comment by ctubbsii** [comment]
@bbux-atg Are you just submitting the pull request and moving on, or are you able to work through the code reviews and update based on feedback?

**Comment by bbux-atg** [comment]
@ctubbsii  I will take ownership and work through any changes that need to be made

**Comment by ctubbsii** [comment]
This PR is very out-of-date, and has merge conflicts. @bbux-atg are you still interested in getting this ready to merge?

**Comment by ctubbsii** [comment]
This PR is out of date and the contributor is not responsive. I'm closing this for now. Please re-open or submit a new PR if this is still a desired feature.

In addition, since the dynamic classloader stuff began being phased out in 2.x, I'm inclined to suggest that we should go a different route here... perhaps through the use of a client-side custom classloader that the user can optionally add to their execution of the shell. If somebody is considering revisiting this, please consider starting a discussion on the mailing list first, so that the efforts to revisit this align with the current direction of the project regarding class loading features.

**Pull request closed without merging**